src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -922,55 +922,17 @@ because xbuild doesn't support framework reference assemblies.
 	</GetAppSettingsDirectory>
 
 	<!-- ensure a version of paths with trailing slashes even if overridden by /p:foo=bar -->
-	<CreateProperty Value="$(AppSettingsDirectory)">
-		<Output TaskParameter="Value" PropertyName="_AppSettingsDirectory"/>
-	</CreateProperty>
-	<CreateProperty Value="$(_AppSettingsDirectory)\">
-		<Output TaskParameter="Value" PropertyName="_AppSettingsDirectory"
-			Condition="!HasTrailingSlash('$(_AppSettingsDirectory)')" />
-	</CreateProperty>
-
-	<CreateProperty Value="$(_AppSettingsDirectory)debug.keystore">
-		<Output TaskParameter="Value" PropertyName="_ApkDebugKeyStore"
-			Condition="'$(_ApkDebugKeyStore)' == ''"
-		/>
-	</CreateProperty>
-
-	<CreateProperty Value="$(MonoAndroidToolsDirectory)">
-		<Output TaskParameter="Value" PropertyName="_MonoAndroidToolsDirectory"/>
-	</CreateProperty>
-	<CreateProperty Value="$(_MonoAndroidToolsDirectory)\">
-		<Output TaskParameter="Value" PropertyName="_MonoAndroidToolsDirectory"
-			Condition="!HasTrailingSlash('$(_MonoAndroidToolsDirectory)')" />
-	</CreateProperty>
-
-	<CreateProperty Value="$(_AndroidNdkDirectory)\">
-		<Output TaskParameter="Value" PropertyName="_AndroidNdkDirectory"
-			Condition="!HasTrailingSlash('$(_AndroidNdkDirectory)')" />
-	</CreateProperty>
-
-	<CreateProperty Value="$(_AndroidSdkDirectory)\">
-		<Output TaskParameter="Value" PropertyName="_AndroidSdkDirectory"
-			Condition="!HasTrailingSlash('$(_AndroidSdkDirectory)')" />
-	</CreateProperty>
-
-	<CreateProperty Value="$(AndroidSdkBuildToolsPath)\">
-		<Output TaskParameter="Value" PropertyName="AndroidSdkBuildToolsPath"
-				Condition="!HasTrailingSlash('$(AndroidSdkBuildToolsPath)')"
-		/>
-	</CreateProperty>
-
-	<CreateProperty Value="$(AndroidSdkBuildToolsBinPath)\">
-		<Output TaskParameter="Value" PropertyName="AndroidSdkBuildToolsBinPath"
-				Condition="!HasTrailingSlash('$(AndroidSdkBuildToolsBinPath)')"
-		/>
-	</CreateProperty>
+	<PropertyGroup>
+		<_AppSettingsDirectory>$([MSBuild]::EnsureTrailingSlash($(AppSettingsDirectory)))</_AppSettingsDirectory>
+		<_ApkDebugKeyStore Condition=" '$(_ApkDebugKeyStore)' == '' ">$(_AppSettingsDirectory)debug.keystore</_ApkDebugKeyStore>
+		<_MonoAndroidToolsDirectory>$([MSBuild]::EnsureTrailingSlash($(MonoAndroidToolsDirectory)))</_MonoAndroidToolsDirectory>
```

Comment:
I believe this should be:

```
$([MSBuild]::EnsureTrailingSlash($(_MonoAndroidToolsDirectory)))
```

Note `_` in `_MonoAndroidToolsDirectory`.

Reply:
This one actually looks right, the incoming value from our existing code doesn't have the `_`: https://github.com/xamarin/xamarin-android/blob/5695ecb0958de9cae81e469d5b890ef2745a74bf/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L939-L941

I read through the diff again to double-check... I think they are the same as what we had before?